packages/yoroi-extension/app/api/ada/lib/state-fetch/types.js

```diff
@@ -339,3 +339,20 @@ export type TokenInfoResponse = {|
 |};
 export type PoolInfoFunc = (body: PoolInfoRequest) => Promise<PoolInfoResponse>;
 export type TokenInfoFunc = (body: TokenInfoRequest) => Promise<TokenInfoResponse>;
+
+export type CatalystRoundInfoRequest = {|
+  ...BackendNetworkInfo
+|};
+
+export type CatalystRound = {|
+  +id: number,
+  +registrationStart: string,
+  +registrationEnd: string,
+  +votingStart: string,
+  +votingEnd: string,
+  +votingPowerThreshold: string,
+|};
+
+export type CatalystRoundInfoResponse = {|
+  [key: string]: (CatalystRound | null),
```

Comment:
Here I think it's better to specifically list the allowed keys, instead of just a generic string key type: `currentFund ` and `nextFund `.